plugins/wasi_nn/wasinnfunc.cpp

```diff
@@ -44,6 +59,52 @@ namespace {
   return DeviceName;
 }
 
+#ifdef WASMEDGE_PLUGIN_WASI_NN_BACKEND_TF
+tensorflow::DataType TensorflowTypeMap(const WASINN::TensorType InType) {
+  uint32_t TFInType;
+  switch (InType) {
+  case WASINN::TensorType::F32:
+    TFInType = 1; // TF_FLOAT
```

Comment:
I prefer to use `return static_cast<tensorflow::DataType>(TF_DataType::TF_FLOAT)` here.